include/pipert/ReceiverChannel.h

```diff
@@ -0,0 +1,27 @@
+#ifndef _RECEIVERCHANNEL_H_
+#define _RECEIVERCHANNEL_H_
+
+#include "pipert/Channel.h"
+
+namespace pipert {
+
+template <class T>
+class ReceiverChannel : public Channel<T> {
+ public:
+  ReceiverChannel(ChannelImpl* impl, const UDPConnection& connection);
```

Comment:
I see, so you want to directly parameterized a created channel with an UDPConnection object? In this case, it can still be part of the public API and I was wrong before.